index.js

```diff
@@ -6,6 +6,13 @@ module.exports = (fn, options = {}) => {
 		throw new TypeError(`Expected the first argument to be a function, got \`${typeof fn}\``);
 	}
 
+	const before = (options.before === undefined) ? false : options.before;
+	const after = (options.after === undefined) ? true : options.after;
```

Comment:
I think it's worth using a regular `Object.assign` or `{...}` to define defaults and destructure all the options in one place.

Reply:
Not a big fan of memory allocations without any purpose, but since it's hard to expect this method to be used frequently, we can add allocation I think.

Reply:
You could also use destructuring with defaults in the function parameters so no new objects would be created